doc/api/http2.md

```diff
@@ -34,34 +45,43 @@ server.on('stream', (stream, headers) => {
   stream.end('<h1>Hello World</h1>');
 });
 
-server.listen(80);
+server.listen(8443);
 ```
 
-Note that the above example is an HTTP/2 server that does not support SSL.
-This is significant as most browsers support HTTP/2 only with SSL.
-To make the above server be able to serve content to browsers,
-replace `http2.createServer()` with
-`http2.createSecureServer({key: /* your SSL key */, cert: /* your SSL cert */})`.
+To generate the certificate and key for this example, run:
+
+```bash
+openssl req -newkey rsa:2048 -nodes -sha256 -subj '/CN=localhost' -x509 -keyout localhost-privkey.pem -out localhost-cert.pem
```

Comment:
tiny style nit: Can you wrap at 80 characters? You can use backslashes to escape end-of-lines in bash, i.e.
```
fooprog arg1 arg2 arg3 \
    arg4 arg5
```
is the same as `fooprog arg1 arg2 arg3 arg4 arg5`